package.json

```diff
@@ -12,24 +12,21 @@
     "morgan": "^1.9.0"
   },
   "scripts": {
-    "build": "npm run clean && npm-run-all --parallel build:static build:node",
-    "build:node": "./tasks/build-node-scripts.sh",
-    "build:prod": "NODE_ENV=production npm run build",
-    "build:static": "./tasks/build-static.sh",
     "clean": "rm -rf dist",
-    "lint": "npm-run-all --parallel lint:eslint lint:stylelint",
-    "lint:eslint": "eslint 'src/**/*.js'",
+    "lint:eslint": "eslint src",
     "lint:stylelint": "stylelint 'src/**/*.css'",
-    "serve": "npm run build && npm-run-all --parallel watch server",
-    "start": "node dist/server/server.js",
-    "server": "node ./node_modules/.bin/nodemon node dist/server/server.js",
-    "test": "npm run test:node && npm run test:static",
-    "test:node": "./tasks/test-node.sh",
-    "test:static": "./node_modules/.bin/karma start",
-    "tools:import_products": "./node_modules/.bin/babel-node tools/import_products.js",
-    "watch": "npm-run-all --parallel watch:static watch:node",
-    "watch:node": "./node_modules/.bin/watch 'npm run build:node' src/server src/data",
-    "watch:static": "./node_modules/.bin/watch 'npm run build:static' src/client"
+    "lint": "npm-run-all --continue-on-error lint:eslint lint:stylelint",
+    "build:backend": "./tasks/build-node-scripts.sh",
+    "build:frontend": "./tasks/build-static.sh",
+    "build": "npm-run-all clean build:backend build:frontend",
+    "build:prod": "NODE_ENV=production npm run build",
+    "watch:backend": "nodemon --watch src/data --watch src/server --watch src/services --watch src/templates --exec 'npm-run-all build:backend start'",
+    "watch:frontend": "nodemon --watch src/client --exec 'npm run build:frontend'",
+    "start:dev": "NODE_ENV=development npm-run-all --parallel watch:backend watch:frontend",
+    "start": "node dist/server/server.js"
+  },
+  "nodemonConfig": {
+    "ext": "css,hbs,js,json,png,svg"
```

Comment:
Server should restart on hbs changes I think.

Reply:
Clarifying: this is the list of file extensions being monitored that will result in a build on change.